beacon-chain/db/kv/blocks_test.go

```diff
@@ -353,6 +355,62 @@ func TestStore_DeleteFinalizedBlock(t *testing.T) {
 	require.NoError(t, db.SaveFinalizedCheckpoint(ctx, cp))
 	require.ErrorIs(t, db.DeleteBlock(ctx, root), ErrDeleteJustifiedAndFinalized)
 }
+
+func TestStore_HistoricalDataBeforeSlot(t *testing.T) {
```

Comment:
Can you expand this test to cover the other buckets too? Or at least verify before pruning that all buckets contain data, and that they are empty after pruning. For some buckets (such as the validator ones or things like `blockParentRootIndicesBucket`) it's unclear.

Reply:
Done!